### PR TITLE
Added: Add a Django management command to create new experiments

### PR DESCRIPTION
**Pull request by drikusroor**
This PR adds a Django management command that allows a developer to create new experiments.

* It first asks the user for the experiment name
* Based on that name it will create a new file with the experiment name in snake case (e.g. `experiment/rules/snake_case.py`) based on a template in `experiment/management/commands/templates/experiment.py`
* It will register the experiment to `experiment/rules/__init__.py` (this might have to be removed later, see also #732 )
* Lastly, it will add a basic unit test in `experiment/rules/tests/test_snake_case.py`, based on the template in `experiment/management/commands/templates/test_experiment.py`

Additionally, this pull request adds a script that allows executing Django management commands through Docker, making it easier to execute this specific command.

## Demonstration

https://github.com/Amsterdam-Music-Lab/MUSCLE/assets/8208970/860d6aed-c3bd-48cb-b60c-b640ce71ceb5

### CODE REVIEW

File: backend/experiment/management/commands/createexperiment.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        filename = f"./experiment/rules/{experiment_name_snake_case}.py"
+
+        # Check if the file already exists
+        if os.path.isfile(filename):
+            # Warn the user that the file already exists and ask if they want to overwrite it
+            self.stdout.write(self.style.WARNING(f"File {filename} already exists"))
+            overwrite = input("Do you want to overwrite it? (y/n): ")
+
+            # If the user does not want to overwrite the file, exit the command
+            if overwrite.lower() != 'y':
```
Comment (BeritJanssen) [inline backend/experiment/management/commands/createexperiment.py:36]: I worry a bit that giving an option to overwrite may break existing experiments in the database. I think it's safer not to provide that option.

File: backend/experiment/management/commands/createexperiment.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+        with open(file_path, 'r') as file:
+            lines = file.readlines()
+
+        # Find the line to insert the new import, maintaining alphabetical order
+        import_index = next(i for i, line in enumerate(lines) if line.startswith('from .') and line > new_import)
+        lines.insert(import_index, new_import)
+
+        # Find the line to insert the new dictionary entry, maintaining alphabetical order within the EXPERIMENT_RULES
+        dict_start_index = lines.index("EXPERIMENT_RULES = {\n") + 1  # Start after the opening brace
```
Comment (BeritJanssen) [inline backend/experiment/management/commands/createexperiment.py:71]: NB: I was hoping to supersede the `EXPERIMENT_RULES` dict with the solution proposed in #732

**Review by BeritJanssen — CHANGES_REQUESTED** [review]
Nice idea! I don't think it's wise to let experimenters overwrite existing rules files though. Other than that, this looks great! Oh yeah, a small note: your code formatter doesn't seem to insert empty lines at the end of files. I understand some other code editors may be thrown by missing empty lines (hence Github flags these as well).

File: backend/experiment/management/commands/createexperiment.py (hunk 2 above)
Comment (drikusroor) [reply backend/experiment/management/commands/createexperiment.py:71]: I can make an issue to remove this method after #732 is merged? Or should I remove it here first?

**Review by BeritJanssen — APPROVED** [review]

**Pull request merged**